scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -348,7 +348,7 @@ object Ctor {
 
 @branch trait Mod extends Tree
 object Mod {
-  @ast class Annot(body: Init) extends Mod
+  @ast class Annot(init: Init) extends Mod
```

Comment:
Can we add a `@deprecated def body = init`? I'd like to migrate this automatically with scalafix.